apps/web/components/Section/Section.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Box, ResponsiveSpace } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { useWindowSize } from 'react-use'
+
+type BackgroundBleed = {
+  fromColor: keyof typeof theme.color
+  toColor: keyof typeof theme.color
+  bleedDirection: 'top' | 'bottom'
+  bleedAmount: number
+}
+
+interface SectionProps {
+  as?: string
+  paddingY?: ResponsiveSpace
+  paddingTop?: ResponsiveSpace
+  paddingBottom?: ResponsiveSpace
+  background?: keyof typeof theme.color
+  backgroundBleed?: BackgroundBleed
+}
+
+const Section: React.FC<SectionProps> = ({
+  as = 'section',
+  children,
+  paddingY,
+  paddingTop,
+  paddingBottom,
+  background,
+  backgroundBleed,
+}) => {
+  const { width } = useWindowSize()
+
+  const generateBackgroundBleed = (backgroundBleed: BackgroundBleed) => {
+    //Background bleed is not available on mobile.
+    if (width < theme.breakpoints.md) {
```

Comment:
I'm surprised, shouldn't you get an error for using `backgroundBleed` as a prop and as a function argument?

```suggestion
    if (!backgroundBleed  || width < theme.breakpoints.md) {
```

Reply:
Implemented. Thanks.